cloud-function/src/handlers/proxy-bsa.ts

```diff
@@ -8,20 +8,27 @@ import {
   fetchImage,
 } from "../internal/pong/index.js";
 
+import stageLookup from "../stripe-plans/stage.js";
+import prodLookup from "../stripe-plans/prod.js";
 import * as env from "../env.js";
 
 import { getRequestCountry } from "../utils.js";
+import { ORIGIN_MAIN } from "../env.js";
 
 const { SIGN_SECRET, BSA_ZONE_KEYS } = env;
 
 const coder = new Coder(SIGN_SECRET);
 const handleGet = createPong2GetHandler(BSA_ZONE_KEYS, coder, env);
 const handleClick = createPong2ClickHandler(coder);
 const handleViewed = createPong2ViewedHandler(coder);
+const lookupData =
```

Comment:
Nit: Rename for clarity.

```suggestion
const plusLookup =
```